locksmith/src/controllers/transactionController.js

```diff
@@ -13,33 +12,20 @@ const transactionCreate = async (req, res) => {
     transaction.recipient
   ) {
     try {
-      await Transaction.findOrCreate({
-        where: {
-          transactionHash: transaction.transactionHash,
-        },
-        defaults: {
-          transactionHash: transaction.transactionHash,
-          sender: ethJsUtil.toChecksumAddress(transaction.sender),
-          recipient: ethJsUtil.toChecksumAddress(transaction.recipient),
-        },
-      })
+      await findOrCreateTransaction(transaction)
       res.sendStatus(202)
     } catch (e) {
       res.sendStatus(400)
     }
   }
 }
 
-const transactionGet = async (req, res) => {
-  const sender = ethJsUtil.toChecksumAddress(req.query.sender)
-
-  let transactions = await Transaction.findAll({
-    where: {
-      sender: { [Op.eq]: sender },
-    },
-  })
-
+const transactionsGet = async (req, res) => {
```

Comment:
Is this the best name for this? Are there likely to be other transaction retrieval methods in the future? It's not obvious to me that this gets transactions owned by a sender.

Reply:
So, not a huge fan of the name either but I think this is consistent with the names picked by @akeem previously and with the actual REST URL `/transactions`. I just put a plural because it indicates many transactions ;p